src/Neo/Hardfork.cs

```diff
@@ -16,6 +16,7 @@ public enum Hardfork : byte
         HF_Aspidochelone,
         HF_Basilisk,
         HF_Cockatrice,
-        HF_Domovoi
+        HF_Domovoi,
+        HF_Echidna,
```

Comment:
I'll refactor this code after the merge of https://github.com/neo-project/neo/pull/3454.